recipes/tensorflow-text/meta.yaml

```diff
@@ -28,10 +29,10 @@ requirements:
   - setuptools >=70
   - wheel
   - absl-py
-  - {{ pin_compatible('tensorflow', max_pin='x.x') }}
+  - tensorflow {{ tensorflow_version }}
   run:
   - python
-  - {{ pin_compatible('tensorflow', max_pin='x.x') }}
+  - tensorflow {{ tensorflow_version }}
```

Comment:
this one will be picked up by the tensorflow run export

Reply:
Which one do you want me to change back?

Reply:
```suggestion
```

Reply:
Done.